docs/community_leadership/community_leadership.rst

```diff
@@ -196,29 +165,57 @@ These are individuals who have previously served on the Mautic Community Leaders
      - 2020
      - 2021
    * - Nico Grienauer
-     - Community Team Assistant Lead
+     - Assistant Team Lead, Community
      - 2020
      - 
-   * - Andy Towne
-     - Mautic Council Member
+   * - Mohit Aghera
+     - Assistant Team Lead, Product (Development)
+     - 2020
+     - 2023
+   * - Norman Pracht
+     - Product Team Lead
+     - 2020
+     - 2023
+   * - Déborah Salves
+     - Assistant Team Lead, Marketing
+     - 2021
+     - 2023
+   * - Joey Keller
```

Comment:
Joey is now Product Team Lead. Although the title is different here, do we still want to list him in the honor roll?

Reply:
I don't know, what do you think is best?

Reply:
sounds fine to me!